### PR TITLE
feat: added reset option to Crosshairs

### PR DESCRIPTION
**Pull request by Ouwen**
Adds the option to reset the annotation. This can be called manually on the ToolGroup toolInstance for reseting purposes.

### CODE REVIEW

File: packages/tools/src/tools/CrosshairsTool.ts [hunk 1]
```diff
@@ -181,6 +181,28 @@ class CrosshairsTool extends AnnotationTool {
       defaultReferenceLineSlabThicknessControlsOn;
   }
 
+  resetAnnotations = () => {
```
Comment (sedghi) [inline packages/tools/src/tools/CrosshairsTool.ts:184]: can't you disable and enable again the tool?

**Review by wayfarer3130 — CHANGES_REQUESTED** [review]
Could you add example buttons to the crosshairs example tools so that we can see the behaviour when you turn the crosshairs off/on again, and a button to specifically reset everything?
It just isn't quite clear how this would get used in practice.

File: packages/tools/src/tools/CrosshairsTool.ts (hunk 1 above)
Comment (wayfarer3130) [reply packages/tools/src/tools/CrosshairsTool.ts:184]: At least add an example with resetting in the tool samples, but also try enable/disable to see what happens.

**Comment by sedghi** [comment]
We have added this in main, thanks!

**Pull request closed without merging**